### DISCUSSION TITLE
Teleport connect license

### DISCUSSION THREAD

**Discussion by Robert904** [gravitational/teleport#37742] [Q&A] [answered]
Hi,
In relation to question https://github.com/gravitational/teleport/discussions/37081 where it is mentioned that Teleport Connect rely on "some private enterprise dependencies", could you confirm the license of the version that can be downloaded on the website ?

Having checked on the website, the source code and the downloaded distribution itself, I can't find this information...
Many thanks

**Comment by webvictim** [comment, marked as answer]
Answered here: https://github.com/gravitational/teleport/discussions/37081#discussioncomment-8398576
